src/app/initial-html.js

```diff
@@ -7,198 +7,151 @@ export default `
 <!-- wp:image -->
 <figure class="wp-block-image"><img alt=""/></figure>
 <!-- /wp:image -->
-
```

Comment:
Minor but I'm curious we're removing empty lines here?

Reply:
I saw this in the diff and I meant to change out this  back to the original and forgot to get back to it actually. My IDE keeps including the spaces